scipy/stats/tests/test_mstats_basic.py

```diff
@@ -880,7 +880,7 @@ def test_result_attributes(self):
         check_named_results(res, attributes, ma=True)
 
 
-#TODO: for all ttest functions, add tests with masked array inputs
+# TODO: for all ttest functions, add tests with masked array inputs
```

Comment:
I think we usually want 2 spaces after a `#`

Reply:
One space is more common
```
eights@eights:~/test/scipy$ grep -ri '# TODO' scipy/ | wc
     76     647    6334
eights@eights:~/test/scipy$ grep -ri '#  TODO' scipy/ | wc
      0       0       0
```